code/modules/surgery/organs/internal/lungs/_lungs.dm

```diff
@@ -847,16 +856,13 @@
 	cold_level_2_threshold = 140
 	cold_level_3_threshold = 100
 
-/obj/item/organ/internal/lungs/cybernetic/emp_act(severity)
-	. = ..()
-	if(. & EMP_PROTECT_SELF)
-		return
-	if(!COOLDOWN_FINISHED(src, severe_cooldown)) //So we cant just spam emp to kill people.
-		owner.losebreath += 20
-		COOLDOWN_START(src, severe_cooldown, 30 SECONDS)
-	if(prob(emp_vulnerability/severity)) //Chance of permanent effects
-		organ_flags |= ORGAN_EMP //Starts organ faliure - gonna need replacing soon.
-
+/obj/item/organ/internal/lungs/cybernetic/surplus
+	name = "surplus cybernetic lungs"
+	desc = "Two fragile sacs of air that only barely mimick the function of human lungs. \
```

Comment:
```suggestion
	desc = "You slowly come to the realization that these two pieces of loose plastic with machinery attached are, in fact, somehow supposed to mimic the function of normal lungs. Dear god. \
```

It's mimic, not mimick, and sack would proobably be more accurate, 'sac' is more used for biological glands

Reply:
ok hoorah